build/Microsoft.DotNet.Cli.DependencyVersions.props

```diff
@@ -11,5 +11,6 @@
     <TemplateEngineVersion>1.0.0-beta1-20170202-111</TemplateEngineVersion>
     <TemplateEngineTemplateVersion>1.0.0-beta1-20170206-112</TemplateEngineTemplateVersion>
     <PlatformAbstractionsVersion>1.1.1</PlatformAbstractionsVersion>

```

Comment:
Are we going to get into any trouble mixing and matching PlatformAbstractions 1.1.1 with DependenyModel 1.0.2?

I suppose not, since we already were mixing and matching these versions.